### ISSUE TITLE
Support isolate 1.0.0 in analzyer

### ISSUE THREAD

**Issue by zoechi** [dart-lang/sdk#28536]
The limitation to 0.2.3 causes dependency problems
```
 $ pub deps
test_project 0.1.0
|-- angular2 3.0.0-alpha
|   |-- logging...
|   |-- meta...
|   |-- analyzer 0.29.5
|   |   |-- args...
|   |   |-- charcode...
|   |   |-- crypto...
|   |   |-- glob...
|   |   |-- html...
|   |   |-- meta...
|   |   |-- path...
|   |   |-- yaml...
|   |   |-- isolate 0.2.3
                    ^^^^^
```

**Comment by lrhn** [comment]
There shouldn't be any urgency to updating - version 1.0.0 is just 0.2.3 graduated to a real version number.

**Comment by zoechi** [comment]
@lrhn but it prevents analyzer to be used together with other packages that depend on `analyzer: ^1.0.0`

If 1.0.0 is the same as 0.2.3 then just extending the dependency to `isolate: '=>0.2.3 <2.0.0'` should not cause any problems.

**Comment by devoncarew** [comment]
fixed by https://github.com/dart-lang/sdk/commit/c1a0b121591f778f6640555f5a80f0aa965cb467

**Comment by bwilkerson** [comment]
That fix hasn't yet been published, and if it's important for it to be published soon we need to backport it to the 0.29.x branch.

**Comment by devoncarew** [comment]
We should back-port then and publish; I'm seeing several issues in other repos related to this.

**Comment by bwilkerson** [comment]
The fix has been backported and published as version 0.29.7.
